Resources/public/js/apps/plugins/ez-registerlanguagehelpersplugin.js

```diff
@@ -58,6 +66,26 @@ YUI.add('ez-registerlanguagehelpersplugin', function (Y) {
 
             Y.Handlebars.registerHelper('translate_property', Y.bind(app.translateProperty, app, app.get('localesMap')));
         },
+
+        /**
+         * Registers the `translate` handlebars helper.
+         * The `translate` helper expects a message that can be found in a domain (See Symfony translation documentation).
+         * A list of variables to be inserted into the translated message can be provided in the `variables` attribute
+         * in the shape of a JSON hash, here is a example:
+         * {"name": "John Doe", "age": "42"}
+         *
+         * @method _registerTranslate
+         * @protected
+         */
+        _registerTranslate: function () {
+            Y.Handlebars.registerHelper('translate', function (message, domain, variables) {
```

Comment:
the order of the function parameters is a bit weird, this should be `message, variables, domain` where `variables` is optional.
That way, you can write this like:

```js
Y.Handlebars.registerHelper('translate', function (message, variables, domain) {
    if ( !domain ) {
        domain = variables;
        variables = {};
    } else {
        variables = JSON.parse(variables);
    }
    return Y.eZ.Translator.trans(message, variables, domain);
});
```